python/ql/src/experimental/semmle/python/frameworks/Django.qll

```diff
@@ -341,8 +341,7 @@ private module Django {
 
     override DataFlow::Node getSql() {
       result.asCfgNode() =
-        [node.getArg([0 .. 5]),
-            node.getArgByName(["select", "where", "params", "tables", "order_by", "select_params"])]
```

Comment:
I'm guessing we need to exclude the indexes of the positional arguments corresponding to `select_params` and `params`?

Reply:
Good point.